deps.sh

```diff
@@ -284,11 +284,49 @@ check_macos_pkgs () {
   PACKAGE_INSTALL_CMD=( brew install ${MISSING_FORMULAE[*]} )
 }
 
+check_arch_pkgs () {
+  local REQUIRED_PKGS=(
+    base-devel        # C/C++ compiler, make, etc.
+    curl              # download rustup
+    pkgconf           # build system
```

Comment:
this is part of base-devel, can see https://archlinux.org/packages/core/any/base-devel/